docs/code-formatting.md

```
@@ -0,0 +1,102 @@
+
+O projeto inclui um arquivo `.editorconfig` para manter estilos de codificação consistentes entre diferentes editores e IDEs. As principais configurações globais são:
+
+- **Charset:** `utf-8` (codificação de caracteres).
+- **End of Line:** `lf` (estilo Unix de quebra de linha).
+- **Indentação:** Espaços (`space`) com tamanho de 2 (`indent_size = 2`).
+- **Insert Final Newline:** `true` (Adiciona uma linha em branco no final de cada arquivo).
+- **Trim Trailing Whitespace:** `true` (Remove espaços em branco no final das linhas).
+
+## Comandos Úteis
+
+#### Verificar erros de Lint
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)